packages/grafana-prometheus/package.json

```diff
@@ -5,7 +5,9 @@
   "version": "12.1.0-pre",
   "description": "Grafana Prometheus Library",
   "keywords": [
-    "typescript"
+    "typescript",
+    "prometheus",
```

Comment:
Is this change relevant to the PR?

Reply:
Yep, this change is for testing the PR as without a change in `packages/**` the levitate workflow won't run. I'll revert it and likely a breaking change dummy commit once I can prove this PR fixes the issue. 👍